src/utils.js

```diff
@@ -72,8 +74,16 @@ const version = () => {
 	console.log( `Wikipedia Preview - version ${APP_VERSION} (${GIT_HASH})` )
 }
 
+const getPreviewHtml = ( title, lang ) => {
+	return new Promise( ( resolve )=> {
```

Comment:
Until we officially remove IE11 support from this product, we can't use promises. You can use a callback like we do in the API layer.